internal/db/queries.sql

```diff
@@ -96,3 +96,7 @@ INSERT INTO mergestat.repo_sync_logs (repo_sync_queue_id, log_type, message)
 SELECT id, 'ERROR', 'No response from job within reasonable interval. Timing out.' FROM timed_out_sync_jobs
 RETURNING repo_sync_queue_id
 ;
+
+-- name: DeleteRemovedRepos :exec 
+DELETE FROM public.repos WHERE repo NOT IN (@currentRepos::string[]);
```

Comment:
```suggestion
DELETE FROM public.repos WHERE NOT (repo = ANY (@currentRepos::text[]));
```

not totally sure (will need to test locally) but try this?